web/src/views/Home/Hero.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import { Button } from "~/components";
+
+const Hero = () => (
+  <main className="relative h-80 md:h-96">
+    <Image
+      fill
+      className="grayscale absolute z-0"
+      src="/images/home-hero-1.png"
+      alt="Comunidad de MakerLab"
+      style={{
+        objectFit: "cover"
+      }}
```

Comment:
Could we use the [`object-fit`](https://tailwindcss.com/docs/object-fit) class instead?
```suggestion
      className="grayscale absolute z-0 object-cover"
      src="/images/home-hero-1.png"
      alt="Comunidad de MakerLab"
```